src/ExceptionInterface.php

```diff
@@ -0,0 +1,21 @@
+<?php
+declare(strict_types=1);
+
+/*
+ * This file is part of the Sonata Project package.
+ *
+ * (c) Thomas Rabaix <thomas.rabaix@sonata-project.org>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+namespace Google\Authenticator;
+
+/**
+ * Indicates the interface is coming from this package.
+ *
+ * @author Iltar van der Berg <kjarli@gmail.com>
+ */
+interface ExceptionInterface
```

Comment:
Actually, there's no need to catch those exceptions as before that, the generation would've failed and there's no way to get around it other than removing the invalid values manually :thinking: